modules/loader/src/main/java/com/opengamma/strata/loader/csv/QuotesCsvLoader.java

```diff
@@ -0,0 +1,133 @@
+/**
+ * Copyright (C) 2015 - present by OpenGamma Inc. and the OpenGamma group of companies
+ *
+ * Please see distribution for license.
+ */
+package com.opengamma.strata.loader.csv;
+
+import java.time.LocalDate;
+import java.util.Arrays;
+import java.util.Collection;
+
+import com.google.common.collect.ImmutableMap;
+import com.opengamma.strata.basics.market.FieldName;
+import com.opengamma.strata.basics.market.MarketDataFeed;
+import com.opengamma.strata.basics.market.ObservableId;
+import com.opengamma.strata.collect.Messages;
+import com.opengamma.strata.collect.id.StandardId;
+import com.opengamma.strata.collect.io.CsvFile;
+import com.opengamma.strata.collect.io.ResourceLocator;
+import com.opengamma.strata.market.id.QuoteId;
+
+/**
+ * Loads a set of quotes into memory from CSV resources.
+ * <p>
+ * The quotes are expected to be in a CSV format, with the following header row:<br />
+ * {@code Date, Scheme, Ticker, Value}.
```

Comment:
The headers in the comment are out of date. It's `Valuation Date` and `Symbology`